resources/opa/exporter-config.yaml

```diff
@@ -0,0 +1,36 @@
+---
+apiVersion: v1
+kind: ExporterConfig
+app: "Open Policy Agent"
+version: 1.0.0
+appVersion: 
+  - '0.20.4'
+maintainers:
+  - name: Sysdig team
+    link: https://sysdig.com/
+description: |
+  # Installing the exporter
+  Gatekeeper exposes a Prometheus metrics endpoint to provide metrics for monitoring service health and performance. Open Policy Agent also exposes a metrics endpoint when running as a server, but since Gatekeeper embeds OPA and we don’t run it as a server, we won’t be using it.
+
+  When installing Gatekeeper, you must either edit the Gatekeeper controller manager deployment or the service with the appropriate annotations for Prometheus scraping
+
+  ```
+  kubectl -n gatekeeper-system patch deploy gatekeeper-controller-manager -p '{"spec":{"template":{"metadata":{"annotations":{"prometheus.io/scrape": "true", "prometheus.io/port": "8888"}}}}}'
+  kubectl -n gatekeeper-system patch deploy gatekeeper-audit -p '{"spec":{"template":{"metadata":{"annotations":{"prometheus.io/scrape": "true", "prometheus.io/port": "8888"}}}}}'
+  ```
+
+  # Sysdig Agent configuration
+
+  Then, if you have configured your Prometheus server or your [Sysdig agent for Prometheus metric collection](https://docs.sysdig.com/en/integrate-prometheus-metrics.html), you should be already ingesting your Gatekeeper metrics.
+
+data: |
```

Comment:
This should be inside a confiurations elemens:
- configurations:
  - name: configuration.yaml
  - data: |
    template:
      metadata:
        annotations:
          container.seccomp.security.alpha.kubernetes.io/manager: runtime/default
          prometheus.io/port: "8888"
          prometheus.io/scrape: "true"